plugins/lookup/binary_file.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2025, Felix Fontein <felix@fontein.de>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+DOCUMENTATION = r"""
+name: binary_file
+author: Felix Fontein (@felixfontein)
+short_description: Read binary file and return it Base64 encoded
+version_added: 11.2.0
+description:
+  - This lookup returns the contents from a file on the Ansible controller's file system.
+  - The file is read as a binary file and its contents are returned Base64 encoded.
+    This is similar to using P(ansible.builtin.file#lookup) combined with P(ansible.builtin.b64encode#filter),
+    except that P(ansible.builtin.file#lookup) does not support binary files as it interprets the contents as UTF-8,
+    which can cause the wrong content being Base64 encoded. It also will print a deprecation message since ansible-core 2.19.
```

Comment:
This deprecation comment seems a bit off. 

It seems to be about a.b.file, in which case I would argue we should not add statements about the deprecation status of some other plugin in these docs. 

OTOH, if we do mention that deprecation without telling the reason why it is deprecated, it feels like teasing, and if we do tell the reason, it probably becomes too lengthy and reinforces the previous point that it should not be mentioned here.

I'd rather not.

Reply:
The ansbile.builtlin.file lookup deprecated handling of binary files. I'll remove this sentence.

Reply:
It wasn't clear to me that _that behaviour_ was deprecated.